pages/room/[id].js

```diff
@@ -87,7 +88,33 @@ export default function Room(props) {
         }
       }
     }
-  }, [data, error, loading, users])
+  }, [data, error, loading])
+
+  const userSubscription = useSubscription(USER_SUBSCRIPTION, {
+    variables: { room: props.roomId },
+  })
+
+  useEffect(() => {
+    if (userSubscription.loading) {
+      //Do nothing
+    }
+    if (userSubscription.error) {
+      //TODO: Handle error
+    }
+    if (userSubscription.data) {
+      if (users.some((e) => e.id === userSubscription.data.userModified.id)) {
```

Comment:
I don't think we need to use two array functions to do the same thing here.

Suggestion: Declare a `const` before the `if` on line 105. This const will `find()` the user. If the `const` variable is undefined, user is not found. else, the user object will be populated in the `const`.

Thoughts?